src/protocol.cpp

```diff
@@ -196,6 +196,7 @@ static std::string serviceFlagToStr(size_t bit)
     case NODE_WITNESS:         return "WITNESS";
     case NODE_COMPACT_FILTERS: return "COMPACT_FILTERS";
     case NODE_NETWORK_LIMITED: return "NETWORK_LIMITED";
+    case NODE_REPLACE_BY_FEE:  return "FULL_RBF";
```

Comment:
```suggestion
    case NODE_REPLACE_BY_FEE:  return "REPLACE_BY_FEE?";
```

Reply:
It's also fine to call this service bit `NODE_FULL_RBF`. The `NODE_REPLACE_BY_FEE` name comes from code so old that the terminology hadn't been settled yet and opt-in-RBF didn't exist.

Reply:
Renamed to `NODE_FULL_RBF`.